### PR TITLE
Developer Experience improvements

### PR DESCRIPTION
**Pull request by GalloDaSballo**
Basic `test_shutdown` implementation

Can now use `.env` file for environment variables
Also updated `README` to reflect that

### CODE REVIEW

File: brownie_hooks.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (poolpitako) [inline brownie_hooks.py:3]: How are you using this file? are you calling 
`python brownie_hooks.py` before working on the repo?

Comment (GalloDaSballo) [reply brownie_hooks.py:3]: It's automatically used by brownie and makes it so that it picks up the .env file

Comment (GalloDaSballo) [reply brownie_hooks.py:3]: If you end up using `export`, the ones you exported end up taking precedence, making it a optional use (i.e. you can continue using your old setup with no changes)

Comment (poolpitako) [reply brownie_hooks.py:3]: TIL

https://github.com/eth-brownie/brownie/blob/f4ba974488b67d1f5ad4acfffc8603af912c31d4/brownie/project/main.py#L946-L949

**Review by poolpitako — APPROVED** [review]

**Comment by poolpitako** [comment]
@GalloDaSballo thanks for the contrib!

**Comment by GalloDaSballo** [comment]
Glad I could help!

**Comment by pandadefi** [comment]
This wasn't linted properly and master is broken

**Comment by GalloDaSballo** [comment]
Addressed in #60 , linting should pass there

**Pull request merged**